src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day_mon calendar--month-length_28">
+      <div class="calendar__day-1"></div>
+      <div class="calendar__day-2"></div>
+      <div class="calendar__day-3"></div>
+      <div class="calendar__day-4"></div>
+      <div class="calendar__day-5"></div>
+      <div class="calendar__day-6"></div>
+      <div class="calendar__day-7"></div>
+      <div class="calendar__day-8"></div>
+      <div class="calendar__day-9"></div>
+      <div class="calendar__day-10"></div>
+      <div class="calendar__day-11"></div>
+      <div class="calendar__day-12"></div>
+      <div class="calendar__day-13"></div>
+      <div class="calendar__day-14"></div>
+      <div class="calendar__day-15"></div>
+      <div class="calendar__day-16"></div>
+      <div class="calendar__day-17"></div>
+      <div class="calendar__day-18"></div>
+      <div class="calendar__day-19"></div>
+      <div class="calendar__day-20"></div>
+      <div class="calendar__day-21"></div>
+      <div class="calendar__day-22"></div>
+      <div class="calendar__day-23"></div>
+      <div class="calendar__day-24"></div>
+      <div class="calendar__day-25"></div>
+      <div class="calendar__day-26"></div>
+      <div class="calendar__day-27"></div>
+      <div class="calendar__day-28"></div>
+      <div class="calendar__day-29"></div>
+      <div class="calendar__day-30"></div>
+      <div class="calendar__day-31"></div>
```

Comment:
The calendar includes 31 days, regardless of the month length. Consider adding a mechanism to set the number of days dynamically.

Reply:
I don't know how to connect number of <li>'s in HTML, with the class name in which I set the month length. I tried to set only 28 <li>'s and then add some more with ::after pseudoclass, but it doesn't work with more than one day ;/ I don't know how to do that. Maybe I don't fully understand what I have to do, but I've tried my best :P